src/b3d/chisight/gen3d/pixel_kernels/pixel_depth_kernels.py

```diff
@@ -19,9 +19,16 @@
 
 
 @Pytree.dataclass
-class PixelDepthDistribution(genjax.ExactDensity):
+class PixelDepthDistributionForDepthReturn(genjax.ExactDensity):
```

Comment:
Renamed this to emphasize that the subclasses of this we have do not have the capacity to return depth nonreturns.